tests/rptest/tests/full_disk_test.py

```diff
@@ -288,7 +288,7 @@ def observed_data_size(pred):
             return pred(observed_total)
 
         # write around 30 megabytes into the topic
-        produce_total_bytes(self.redpanda, self.topic, nbytes(30))
+        produce_total_bytes(self.redpanda, self.topic, nbytes(100))
```

Comment:
We could also use `produce_until_segments()` from `util.py` to be sure we have `>= 2` segments

Reply:
I'm inclined to leave it as is. produce_until_segments doesn't have a variant where it would ensure n>=2 for each partition.